<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MrvIo</class>
 <widget class="QWidget" name="MrvIo">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>400</width>
    <height>300</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <widget class="QGroupBox" name="groupBox">
     <property name="title">
      <string/>
     </property>
     <layout class="QGridLayout" name="gridLayout">
      <item row="0" column="0">
       <widget class="QLabel" name="label">
        <property name="text">
         <string>Mode</string>
        </property>
       </widget>
      </item>
      <item row="0" column="1">
       <widget class="QComboBox" name="cmbMode">
        <item>
         <property name="text">
          <string>UART</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>SWITCH</string>
         </property>
        </item>
       </widget>
      </item>
      <item row="1" column="0" colspan="2">
       <widget class="QTabWidget" name="tabWidget">
        <property name="currentIndex">
         <number>0</number>
        </property>
        <widget class="MrqUart" name="pageUart">
         <attribute name="title">
          <string>UART</string>
         </attribute>
        </widget>
        <widget class="MrvSwitch" name="pageSwitch">
         <attribute name="title">
          <string>Switch</string>
         </attribute>
        </widget>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <spacer name="verticalSpacer">
     <property name="orientation">
      <enum>Qt::Vertical</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>20</width>
       <height>40</height>
      </size>
     </property>
    </spacer>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>MrqUart</class>
   <extends>QWidget</extends>
   <header>mrquart.h</header>
   <container>1</container>
  </customwidget>
  <customwidget>
   <class>MrvSwitch</class>
   <extends>QWidget</extends>
   <header>mrvswitch.h</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
